core/unit_test/TestViewAPI_d.hpp

```diff
@@ -53,4 +53,8 @@ TEST_F(TEST_CATEGORY, view_api_d) {
   TestViewAPI<double, TEST_EXECSPACE>::run_test_view_operator_c();
 }
 
+TEST_F(TEST_CATEGORY, view_allocation_error) {
+  TestViewAPI<double, TEST_EXECSPACE>::run_test_error();
+}
```

Comment:
@masterleinad sorry about that...
#2450 will need to be updated so replace `TEST_F -> TEST`

Reply:
Not a problem.